safehttp/mux.go

```diff
@@ -204,3 +205,17 @@ func (s *ServeMuxConfig) Clone() *ServeMuxConfig {
 	copy(c.interceptors, s.interceptors)
 	return c
 }
+
+// Handler returns the handler to use for the given method and path.
+// TODO: add a host parameter?
+func (m *ServeMux) Handler(method, path string) http.Handler {
+	r := &http.Request{
+		Method: method,
+		Host:   "",
+		URL: &url.URL{
+			Path: path,
+		},
+	}
```

Comment:
Yeah, this is a bit hacky.

My recommendation:

```
func (m *ServeMux) RawHandler(path string) http.Handler {
  return m.handlers[path]
}
```

You'd need to add the `handlers` field to `ServeMux`, which shouldn't be a problem.

Reply:
There is one problem with that:

Requesting a handler for "/foo/bar" will not return the handler for "/foo/", but just the exact one if one is registered. Otherwise it will return the 404 handler.

So what about using the ServeMux as a factory that takes a safe handler and returns a standard handler wrapped with all the security stuff? We can even rename the `Handler` method to something more clearly stating this is to be used just for transition.

Reply:
Oh, good catch!

Maybe `RegisteredHandler` would be enough here? We can godoc that this is supposed to be used for transition purposes only too.

Reply:
Not a big fan of the name TBH, what about WrappedHandlerDuringTransition or something similarly communicative?